api/wasm/cpp/proxy_wasm_intrinsics.cc

```diff
@@ -0,0 +1,110 @@
+// NOLINT(namespace-envoy)
```

Comment:
Is the "NOLINT" used by build system?

Reply:
./tools/check_format.py will complain if the namespace is missing, but since this is for the API, it is essentially external code.